page-objects/createTeam.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
 	},
 	addTwoTeamMemebers: async function(teammember1, teammember2) {
 		await this.addTeamMembersSteps(teammember1);
-		await this.addTeamMembersSteps(teammember2);
+		// await this.addTeamMembersSteps(teammember2);
```

Comment:
If everything works without this step, can we remove it then? Or is currently only one user added? Or we also put a comment here, like I suggested below.

Reply:
why do you reduce a number of members?

Reply:
A new feature was developed so that the teachers loose the ability to create students. So now we can only add students, that are already existing. Not sure though, if we have one or two students.

Reply:
done